src/main/scala/firrtl/Utils.scala

```diff
@@ -165,7 +165,7 @@ object Utils extends LazyLogging {
     * @param message - possible string to emit,
     * @param exception - possible exception triggering the error.
     */
-  def throwInternalError(message: String = "", exception: Option[Exception] = None) = {
+  def throwInternalError(message: String = "", exception: Option[Throwable] = None) = {
```

Comment:
Note that this looks spooky and not binary compatible, but I tested it an it is binary compatible due to type erasure (since the real type here is `Option`). MiMa did not report any issues and I also manually checked in a project that compiled with firrtl 1.5.4 and ran with firrtl from this branch.